pallets/msa/src/lib.rs

```diff
@@ -1292,6 +1360,87 @@ impl<T: Config> Pallet<T> {
 		block_number / (T::BlockNumber::from(T::MortalityWindowSize::get())) %
 			T::BlockNumber::from(T::NumberOfBuckets::get())
 	}
+
+	/// Get list of account ids that are associated with a given msa
+	pub fn get_msa_keys_offchain(msa_id: MessageSourceId) -> (T::BlockNumber, Vec<T::AccountId>) {
+		let msa_keys_res = get_msa_keys::<MessageSourceId, T::AccountId, T::BlockNumber>(msa_id);
+		if let Err(e) = msa_keys_res {
+			log_err!("Error getting msa keys from offchain storage: {:?}", e);
+			return (T::BlockNumber::zero(), vec![])
+		}
+		msa_keys_res.unwrap()
```

Comment:
Use of `unwrap()` without a test is unsafe.

Reply:
I intend to add tests but yes 1367 will ensure error is caught and since get_msa_keys<> does decoding we are sure it will be either pass or fail